test/functional/feature_versionbits_warning.py

```diff
@@ -91,14 +90,14 @@ def run_test(self):
 
         # Generating one block guarantees that we'll get out of IBD
         node.generatetoaddress(1, node_deterministic_address)
-        wait_until(lambda: not node.getblockchaininfo()['initialblockdownload'], timeout=10, lock=mininode_lock)
+        self.wait_until(lambda: not node.getblockchaininfo()['initialblockdownload'], timeout=10, lock=mininode_lock)
```

Comment:
```suggestion
        self.wait_until(lambda: not node.getblockchaininfo()['initialblockdownload'], timeout=10)
```

why the lock?

Reply:
The class of this method inherits from `BitcoinTestFramework` which does not have a default lock.

Reply:
but why is the lock needed? Why does it need to be taken here?

Reply:
I frankly don't know. It was there before, so I didn't touch it.